src/rewriter/node.py

```diff
@@ -192,12 +237,16 @@ def __init__(self, base, args=None, is_list=False, is_const=False):
         self.is_list = is_list
         self.is_const = is_const
 
+        if base == BaseSort.BitVec and not self.children[0].isnumeric():
+            die("Non-numeric bitvector size is not supported: {self.children[0]}")
```

Comment:
```suggestion
            die("Non-numeric bit-vector size is not supported: {self.children[0]}")
```

Make sure to be consistent with existing code in how to write bit-vectors.

Reply:
fixed